nx.test.json

```diff
@@ -0,0 +1,27 @@
+{
```

Comment:
Could you explain why there is a separate `test` config? It seems to be for publishing the whole monorepo, which we do not want to do?

Reply:
This is only used for integration tests. I've added docs: https://github.com/microsoft/react-native-macos/blob/b873a8eea01e8efea884fd66c6859ed230272824/docs/Releases.md#configuration